### DISCUSSION TITLE
[AnalyticalTable] Select All when grouped

### DISCUSSION THREAD

**Discussion by SpartanLoco91** [UI5/webcomponents-react#5929] [Q&A]
Hi to all, it's possible, using the AnalyticalTable, when grouped, clicking the select all checkbox, select only the row in the expanded group?
Thank you very much

**Comment by Lukas742** [comment]
Hi @SpartanLoco91 

this behavior is not supported out of the box, but can probably be achieved via a [custom plugin hook](https://sap.github.io/ui5-webcomponents-react/?path=/docs/data-display-analyticaltable-recipes--docs#how-to-add-your-own-plugin-hooks). Unfortunately I currently don't have the time to look into this more, but feel free to try it out yourself. 

As reference you can for example take a look at our implementations of custom plugin hooks, which you can find here: https://github.com/SAP/ui5-webcomponents-react/tree/main/packages/main/src/components/AnalyticalTable/pluginHooks
